rasa/nlu/tokenizers/convert_tokenizer.py

```diff
@@ -30,16 +39,120 @@ class ConveRTTokenizer(WhitespaceTokenizer):
         "intent_split_symbol": "_",
         # Regular expression to detect tokens
         "token_pattern": None,
-        # Remote URL of hosted model
-        "model_url": TF_HUB_MODULE_URL,
+        # Remote URL/Local path to model files
+        "model_url": None,
     }
 
-    def __init__(self, component_config: Dict[Text, Any] = None) -> None:
-        """Construct a new tokenizer using the WhitespaceTokenizer framework."""
+    def _validate_model_files_exist(self, model_directory: Text) -> None:
+        """Check if essential model files exist inside the model_directory.
 
+        Args:
+            model_directory: Directory to investigate
+        """
+        files_to_check = [
+            os.path.join(model_directory, "saved_model.pb"),
+            os.path.join(model_directory, "variables/variables.index"),
+            os.path.join(model_directory, "variables/variables.data-00001-of-00002"),
+            os.path.join(model_directory, "variables/variables.data-00000-of-00002"),
+        ]
+
+        for file_path in files_to_check:
+            if not os.path.exists(file_path):
+                raise RasaException(
+                    f"""File {file_path} does not exist.
+                    Re-check the files inside the directory {model_directory}.
+                    It should contain the following model files - {files_to_check}"""
```

Comment:
should we concatenate the list with `,` instead of printing the pure python list? There is also a utility function for it as far as I know